src/TemplateEditor.js

```diff
@@ -615,7 +619,7 @@ export default class TemplateEditor extends React.Component {
     const insertInx = parentControlData.findIndex(
       ({ id }) => id === control.id
     )
-    const deleteLen = parentControlData.length - insertInx - 1
+    const deleteLen = parentControlData.length - insertInx - 4
```

Comment:
Can you please add a comment to explain why there is a hardcoded 4 here?

Reply:
The hardcoded number has been replaced with a labeling method. I just label the inserted cards when inserted, then use the labels to generate a count for how many to replace.
https://github.com/open-cluster-management/temptifly/pull/82/files#diff-86fc06aa59c90aa88dd75d9371c29ff8f70ecc90a24ff24899b6b24bfb004a39R622